panels/user-accounts/data/password-dialog.ui

```diff
@@ -249,6 +249,20 @@
                       </packing>
                     </child>
                     <child>
+                      <object class="GtkRadioButton" id="action-no-password-radio">
+                        <property name="label" translatable="yes">Allow user to login without a password</property>
```

Comment:
"log in" should be two words here since it's used as a verb, unlike "on next login" where it is used as a noun (one word).